.github/workflows/linux.yml

```diff
@@ -0,0 +1,110 @@
+name: github-Linux
+
+on:
+  pull_request:
+    paths-ignore:
+    - '**/*.rst'
+    - '**/*.md'
+    - '**/requirements.txt'
+    - '**/*.py'
+    - 'docs/**'
+    types: [ opened, reopened, synchronize ]
+
+permissions:
+  contents: none
+
+# Cancels any in progress 'workflow' associated with this PR
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  kokkos_version: 4.4.01
+
+jobs:
+  check-pr-labels:
+    runs-on: [ubuntu-latest]
+    steps:
+      - uses: docker://agilepathway/pull-request-label-checker@sha256:ee57b0e1aedab22063ce6467a6e6358e254a9204693ca20d8a16b2d891db8d5f  # v1.6.32
+        with:
+          none_of: 'AT: WIP'
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+  sanitizers:
+    needs: check-pr-labels
+    # TODO: allow re-run via retest label if: ${{ github.event.label.name == 'AT: RETEST' }}
+    name: osx-ci
+    runs-on: [ubuntu-latest]
+
+    strategy:
+      matrix:
+        include:
+          - backend: "SERIAL"
```

Comment:
Do we plan to expand this further? Otherwise maybe the matrix part can be removed and the variable associated with it can be hard coded directly in the configuration options?

Reply:
Maybe a similar build just with clang instead? I think they have different implementations but the same interface of these sanitizers.